tests/test_environment.py

```diff
@@ -35,3 +38,37 @@ def test_environment(environment_path, env):
         traceback.print_exc()
 
     assert no_error_loop
+
+
+@pytest.mark.parametrize('env', ENVS)
+def test_environment2(environment_path, env):
```

Comment:
How about
```
def test_environment_network(environment_path, env):
    """Test battle loop using network battle interface of environment"""
```

Then, the old one should be the 'local' version?
```
def test_environment_local(environment_path, env):
    """Test battle loop using local battle interface of environment"""
```

Reply:
Changing names is good idea.